pkg/cmd/cli/cmd/newbuild.go

```diff
@@ -128,7 +128,7 @@ func RunNewBuild(fullName string, f *clientcmd.Factory, out io.Writer, in io.Rea
 	for _, item := range result.List.Items {
 		switch t := item.(type) {
 		case *buildapi.BuildConfig:
-			fmt.Fprintf(c.Out(), "A build configuration was created - you can run `%s start-build %s` to start it.\n", fullName, t.Name)
+			fmt.Fprintf(c.Out(), "A build configuration was created and an initial build triggered - you can run `%s build-logs %s-1` to stream the logs as the build progresses.\n", fullName, t.Name)
```

Comment:
> Use bc.Status.LatestVersion for this

This won't do the job. `bc.Status.LastVersion` is `0` at that particular reference to the BC, and instead of doing `bc.Status.LastVersion+1` I believe it's better here to hard-code the "-1" suffix.

If you delete the build created by `new-build` and run `oc new-build` again, your first build is once again "mybuildconfigname-1".